### ISSUE TITLE
Cannot  lunch statsgen.py

### ISSUE THREAD

**Issue by Lookya23** [iphelix/pack#22]
Hi !

When i try to start  `py statsgen.py`  i get this error : 

```
  File "C:\Users\xxxx\hashcat\pack\statsgen.py", line 178
    print "[+] Analyzing %d%% (%d/%d) of passwords" % (self.filter_counter*100/self.total_counter, self.filter_counter, self.total_counter)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?
```

I'm beeginer, i'm wrong somewhere or it's bug ?

I have Python 3.10 x64  and hashcat v6.2.4-136

Thanks for your help !

**Comment by spartanhernan** [comment]
youre not wrong going through the same pain, looks like a lot of debugging needs to be done , currently working on getting a running update to this

**Comment by roycewilliams** [comment]
This version still depends on Python 2.x. This fork might match your setup better:

https://github.com/Hydraze/pack

**Comment by spartanhernan** [comment]
FOUND THE SOLUTION TO MAKE IT WORK 


download python 2 and run it as you would python 3 


downloaded py 2 and stored it in a folder called py2  ran the python executable as if it were the command  



D:\TRAINING-GROUNDS\py2>python.exe "D:\TRAINING-GROUNDS\policygen.py"
                       _
     PolicyGen 0.0.2  | |
      _ __   __ _  ___| | _
     | '_ \ / _` |/ __| |/ /
     | |_) | (_| | (__|   <
     | .__/ \__,_|\___|_|\_\
     | |
     |_| iphelix@thesprawl.org
Boom i can now use maskgen
